packages/@okta/vuepress-site/code/react/okta_react_sign-in_widget/index.md

```diff
@@ -247,11 +235,11 @@ class App extends Component {
 export default App;
 ```
 
-Create `src/AppWithRouterAccess.js` and include your project components and routes. `Security` is the component that controls the authentication flows, so it requires your OpenId Connect configuration. By default, `@okta/okta-react` redirects to Okta's login page when the user isn't authenticated. In this example, `onAuthRequired` is overridden to redirect to the custom login route instead:
+Create `src/AppWithRouterAccess.js` and include your project components and routes. `Security` is the component that controls the authentication flows, so it requires your OpenID Connect configuration. By default, `@okta/okta-react` redirects to Okta's login page when the user isn't authenticated. In this example, `onAuthRequired` is overridden to redirect to the custom login route instead:
```

Comment:
....redirects to Okta's login page when.... ->

...redirects to Okta's sign-in page when....

....to the custom login route instead: ->
...to the custom sign-in route instead:

Sorry, I missed a couple

Reply:
I fixed these. Can you please merge when tests pass? I don't have permission.